README.md

```diff
@@ -16,9 +16,10 @@ It has been developed and released by the [Amnesty International Security Lab](h
 
 ## Installation
 
-First you need to install dependencies, on Linux `sudo apt install python3 python3-pip libusb-1.0-0` or on MacOS `brew install python3 libusb`.
+First you need to install dependencies, on Linux `sudo apt install python3 python3-pip libusb-1.0-0` or on MacOS `brew install python3 libusb`. For Arch Linux, `sudo pacman -S python-pip libusb`. 
+
+Then you can install mvt from pypi with `pip install mvt` (use `--user` in case of Arch Linux to avoid conflicts with pacman packages), or directly from sources:
```

Comment:
`pip` might default to python2 on a lot of systems, so `pip3` or probably better `python3 -m pip` should be used.

Reply:
That needs to be fixed. Thank you